src/util/compile-hook.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs-promise';
+import ora from 'ora';
+import path from 'path';
+
+export default async(originalDir, buildPath, electronVersion, pPlatform, pArch, done) => {
+  const compileSpinner = ora.ora('Compiling Application').start();
+
+  const compileCLI = require(path.resolve(originalDir, 'node_modules/electron-compile/lib/cli.js'));
+
+  async function compileAndShim(appDir) {
+    for (const entry of await fs.readdir(appDir)) {
+      if (!entry.match(/^(node_modules|bower_components)$/)) {
+        const fullPath = path.join(appDir, entry);
+
+        if ((await fs.stat(fullPath)).isDirectory()) {
+          const log = console.log;
+          console.log = () => {};
```

Comment:
❤️ this hack....

`electron-compile` logs a warning of `process.env !== 'production'` which messes with the spinner. 😢